lesson02/userdata.sh

```diff
@@ -10,5 +10,5 @@ sudo service docker start
 #All commands here are executed as super admin
 #but still, let's add the ec2-user to the docker group
 usermod -a -G docker ec2-user 
- #Run the nginx 
-docker run -d -p 80:80 --restart=always nginx:${docker_tag}
\ No newline at end of file
+#Run the nginx
```

Comment:
Since this may no longer be nginx, should we modify the comment to something similar to:
"# Run the Docker image"